superset/config.py

```diff
@@ -308,6 +308,8 @@ def _try_json_readsha(  # pylint: disable=unused-argument
     "TAGGING_SYSTEM": False,
     "SQLLAB_BACKEND_PERSISTENCE": False,
     "LISTVIEWS_DEFAULT_CARD_VIEW": False,
+    "DISPLAY_MARKDOWN_HTML": True,
+    "ESCAPE_MARKDOWN_HTML": False,
```

Comment:
Could we add comments here that explain how these affect the behaviour? Also, it could be a good idea to add a comment in `UPDATING.md` in case someone is currently relying on this functionality.

Reply:
The default settings in this PR mimic the current default behavior, so the feature shouldn't require any action when updating.

That said, the comments to seem valuable. I'll add 'em!

Reply:
![image](https://user-images.githubusercontent.com/33317356/96679120-bac04500-137b-11eb-82b5-2f1746fcffd0.png)

Reply:
Added some comments... let me know if they don't make sense.